doc: genai-perf benchmark & slurm multi-node for trtllm-serve doc (#3407)
NVIDIA/TensorRT-LLM: docs/source/commands/trtllm-serve.rst

Inference Endpoints
-------------------

> After you start the server, you can send inference requests through completions API and Chat API, which are compatible with corresponding OpenAI APIs. We use ``TinyLlama-1.1B-Chat-v1.0`` for examples in the following sections.

Reviewer: ```suggestion
After you start the server, you can send inference requests through completions API and Chat API, which are compatible with corresponding OpenAI APIs. We use [``TinyLlama-1.1B-Chat-v1.0``](https://huggingface.co/TinyLlama/TinyLlama-1.1B-Chat-v1.0) for examples in the following sections.
```

Reviewer: Ok, is there a way to link into rst file?

Author: Yes, but I think it is kind of hard to show a piece of text as both code and link, how about using link for the first occurrence of TinyLlama-1.1B-Chat-v1.0 and code for others? I preview locally and the doc looks acceptable.

Reviewer: Sure, sounds good to me, thanks.

    :language: bash
    :linenos:

Benchmark
---------

> You can use any benchmark clients to test the serving performance of ``trtllm_serve``, in the following example, we use ``genai-perf`` as an example.

Reviewer: ```suggestion
You can use any benchmark clients to test the serving performance of ``trtllm-serve``, in the following example, we recommend using ``genai-perf``.
```

Reviewer: Besides, why is "``" used, instead of "`"?

Author: > Besides, why is "``" used, instead of "`"?

No special reason, just follow the previous notation in this doc

Author: Would be weird to say "in the example we recommend", will change the statement though.

Reviewer: Sure, thanks.
